sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -537,7 +537,6 @@ class CodegenContext {
       val funcCode: String =
         s"""
           public int $compareFunc(InternalRow a, InternalRow b) {
-            InternalRow i = null;
```

Comment:
thanks -- but not removing it from here would lead to declaring a local variable `i` that we might not use when `(ordering.size > numberOfComparisonsThreshold) `. what do you think?